images/ceph/Makefile

```diff
@@ -66,7 +66,7 @@ $(YQ):
 	@if [ ! "$(INCLUDE_CSV_TEMPLATES)" = "" ]; then\
 		echo === installing yq $(GOHOST);\
 		mkdir -p $(TOOLS_HOST_DIR)/tmp;\
-		GOPATH=$(TOOLS_HOST_DIR)/tmp GOBIN=$(TOOLS_HOST_DIR) $(GOHOST) get gopkg.in/mikefarah/yq.v2;\
+		GOPATH=$(TOOLS_HOST_DIR)/tmp GOBIN=$(TOOLS_HOST_DIR) GO111MODULE=on $(GOHOST) get -u github.com/mikefarah/yq@3c701fe;\
```

Comment:
Is that the sweet sound of go modules? https://github.com/rook/rook/pull/3793 :wink: